demo/package.json

```diff
@@ -91,8 +91,8 @@
 		"@sveltejs/vite-plugin-svelte": "^3.0.2",
 		"@types/dom-view-transitions": "^1.0.4",
 		"bootstrap-icons": "^1.11.3",
+		"compare-versions": "^6.1.0",
```

Comment:
Shouldn't we use it also in our scripts and remove completely the dependency to `semver` then?